tools/install-jb.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -eu
+
+VERSION=${1:-}
+
+if [ "x√" = "x" ]; then
```

Comment:
This comparison looks pretty broken, are you sure you didn't want

```suggestion
if [ "x${VERSION}" = "x" ]; then
```

Reply:
Yep that is super broken 🙈